nullable/bool.go

```diff
@@ -0,0 +1,59 @@
+package nullable
+
+import (
+	"database/sql"
+	"database/sql/driver"
+	"encoding/json"
+	"reflect"
+)
+
+// Bool defines a nullable bool
+type Bool struct {
+	Bool  bool
+	Valid bool
```

Comment:
Did you consider `IsNull` instead of `Valid` for this field?

Usage would then be `if foo.IsNull {...}` instead of `if foo.Valid {...}`

Reply:
IOW: are nullable types Valid/Invalid or Null/NotNull?

Reply:
The reason why it's called that is because it follows exactly the field names that the go standard library uses for such types. Unless there is a valid reason to change that, I figured sticking to it was probably for the best.

Reply:
👍 in database/sql, someone suggested calling the field NotNull, but that was rejected because people dont like using negative boolean fields